src/queries/getReviewByUser.js

```diff
@@ -0,0 +1,6 @@
+const db = require("../database/db_connection");
+
+const getReviewByUser = id => db.query(
+    `select b.name,r.rating,r.review_body,r.date_created as reviewDate from reviews r join businesses b on r.business_id = b.id join users u on u.id=r.user_id where u.id=$1`, [id]);
```

Comment:
can you change the name of the keys here ? for example ` b.name as businessName`
the same keys that are shown in #33